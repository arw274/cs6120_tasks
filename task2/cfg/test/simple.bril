# 0 -> {}
@main (input: int) {
    ret;
}