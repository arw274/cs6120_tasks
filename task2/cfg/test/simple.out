Block 0: [{'op': 'ret'}]
{'main': {0: []}}
